wg-automl/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# AutoML Working Group
+
+WG AutoML is responsible for all aspects of Automated Machine Learning technologies on Kubeflow.
+
+The [charter](charter.md) defines the scope and governance of the AutoML Working Group.
+
+## Meetings
+* Regular Katib Meeting (Asia & Europe friendly): [Wednesdays at 10:00 am UTC (Coordinated Universal Time)]() (monthly - second Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00 am&tz=UTC%20%28Coordinated%20Universal%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64).
+* Regular Katib Meeting (US friendly): [Wednesdays at 4:00 pm UTC (Coordinated Universal Time)]() (monthly - fourth Wednesday every month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=4:00 pm&tz=UTC%20%28Coordinated%20Universal%20Time%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64).
+
+## Organizers
+
+* Andrey Velichkevich (**[@andreyvelich](https://github.com/andreyvelich)**), Cisco
+* Ce Gao (**[@gaocegege](https://github.com/gaocegege)**), Caicloud
+* Johnu George (**[@johnugeorge](https://github.com/johnugeorge)**), Cisco
+
+## Contact
+- Slack: [#wg-automl](https://kubernetes.slack.com/messages/wg-automl)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubeflow-discuss)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Farea/wg-automl)
+- GitHub Teams:
+    - [@kubernetes/wg-automl-bugs](https://github.com/orgs/kubernetes/teams/wg-automl-bugs) - Bug Triage and Troubleshooting
```

Comment:
Should we change it to `https://github.com/orgs/kubeflow/teams/....` ?

Reply:
I have some leftovers from the templates. Let me remove them